### ISSUE TITLE
Xonda conda 4.5 incompatibility

### ISSUE THREAD

**Issue by CJ-Wright** [gforsyth/xonda#33]
Installing the newest conda broke xonda. I'll try to get a traceback ASAP (I reverted to conda 4.3 so I could continue working)

**Comment by gforsyth** [comment]
Hey @CJ-Wright -- yeah, if you can post a traceback and also your `xonfig` -- fwiw I'm using 4.5.1 and haven't had any problems

**Comment by CJ-Wright** [comment]
```python
Traceback (most recent call last):
  File "/home/christopher/mc/lib/python3.6/site-packages/xonsh/__amalgam__.py", line 20657, in main
    args = premain(argv)
  File "/home/christopher/mc/lib/python3.6/site-packages/xonsh/__amalgam__.py", line 20610, in premain
    env = start_services(shell_kwargs, args)
  File "/home/christopher/mc/lib/python3.6/site-packages/xonsh/__amalgam__.py", line 20559, in start_services
    xonshrc_context(rcfiles=rc, execer=execer, ctx=ctx, env=env, login=login)
  File "/home/christopher/mc/lib/python3.6/site-packages/xonsh/__amalgam__.py", line 16142, in xonshrc_context
    login=login)
  File "/home/christopher/mc/lib/python3.6/site-packages/xonsh/__amalgam__.py", line 16198, in static_config_run_control
    update_context(name, ctx=ctx)
  File "/home/christopher/mc/lib/python3.6/site-packages/xonsh/__amalgam__.py", line 9103, in update_context
    modctx = xontrib_context(name)
  File "/home/christopher/mc/lib/python3.6/site-packages/xonsh/__amalgam__.py", line 9070, in xontrib_context
    m = importlib.import_module(spec.name)
  File "/home/christopher/mc/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/christopher/mc/lib/python3.6/site-packages/xontrib/xonda.xsh", line 4, in <module>
    from conda import config
ImportError: cannot import name 'config'
Xonsh encountered an issue during launch
Failback to /bin/sh
```
```
+------------------+-----------------+
| xonsh            | 0.6.0           |
| Python           | 3.6.3           |
| PLY              | 3.9             |
| have readline    | True            |
| prompt toolkit   | 1.0.15          |
| shell type       | prompt_toolkit  |
| pygments         | 2.2.0           |
| on posix         | True            |
| on linux         | True            |
| distro           | debian          |
| on darwin        | False           |
| on windows       | False           |
| on cygwin        | False           |
| is superuser     | False           |
| default encoding | utf-8           |
| xonsh encoding   | utf-8           |
| encoding errors  | surrogateescape |
+------------------+-----------------+
```

**Comment by gforsyth** [comment]
Can you try upgrading `xonda`?  I believe this issue was fixed in 0.2.5

**Comment by CJ-Wright** [comment]
Awesome thank you very much!
